ra/ra.go

```diff
@@ -231,7 +231,7 @@ func NewRegistrationAuthorityImpl(
 		maxContactsPerReg:            maxContactsPerReg,
 		keyPolicy:                    keyPolicy,
 		maxNames:                     maxNames,
-		reuseValidAuthz:              reuseValidAuthz,
+		reuseValidAuthz:              true,
```

Comment:
I understand that we plan to do a followup PR, lets make sure we add a `TODO(#2734)` comment above this hardcoded bool so we understand that this is something we plan to address.